code/game/machinery/requests_console.dm

```diff
@@ -267,6 +267,7 @@ GLOBAL_LIST_EMPTY(req_console_ckey_departments)
 		GLOB.news_network.SubmitArticle(message, department, "Station Announcements", null)
 		usr.log_talk(message, LOG_SAY, tag="station announcement from [src]")
 		message_admins("[ADMIN_LOOKUPFLW(usr)] has made a station announcement from [src] at [AREACOORD(usr)].")
+		deadchat_broadcast(" made a station announcement from <span class='name'>[get_area_name(usr, TRUE)]</span>.", "<span class='name'>[usr.real_name]</span>", usr)

```

Comment:
`" made a station announcement`
I think you're missing the name

Reply:
all the other deadchat_broadcasts are patterned like this, do a search

Reply:
That's a really ugly syntax but you're right